proto/ibc/applications/fee/v1/fee.proto

```diff
@@ -10,15 +10,14 @@ option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
 // See Fee Payment Middleware spec:
 // https://github.com/cosmos/ibc/tree/master/spec/app/ics-029-fee-payment#fee-middleware-contract
 message Fee {
-  repeated cosmos.base.v1beta1.Coin amount = 1
-      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  cosmos.base.v1beta1.Coin receive_fee = 1 [(gogoproto.moretags) = "yaml:\"receive_fee\""];
```

Comment:
Should these be `sdk.Coins` instead of `Coin`? cc: @colin-axner

Reply:
I was wondering this myself. Unsure what is the best practice here.